src/steps/abort_merge_branch_step.go

```diff
@@ -1,13 +1,16 @@
 package steps
 
-import "github.com/git-town/git-town/src/git"
+import (
+	"github.com/git-town/git-town/src/drivers"
+	"github.com/git-town/git-town/src/git"
+)
 
 // AbortMergeBranchStep aborts the current merge conflict.
 type AbortMergeBranchStep struct {
 	NoOpStep
 }
 
 // Run executes this step.
-func (step *AbortMergeBranchStep) Run(repo *git.ProdRepo) error {
+func (step *AbortMergeBranchStep) Run(repo *git.ProdRepo, driver drivers.CodeHostingDriver) error {
```

Comment:
I'm not too big of a fan of providing the drivers to every step like this when they are used only in some, but it's the most straightforward way that is easiest to understand. If we inject more dependencies in the future, we can refactor this into a `context` struct.

Reply:
Conceptually I like the idea to introduce the driver at the very start of every command. That way it's always around with you for present and _future_ use cases.